kythe/go/extractors/golang/golang.go

```diff
@@ -200,21 +200,38 @@ func (e *Extractor) dirToImport(dir string) (string, error) {
 // If the package has already been located, its existing package is returned.
```

Comment:
does the comment up here need updating at all?

Reply:
I added a note about multiple packages being resolved.